src/lib/Flasher.js

```diff
@@ -351,7 +359,7 @@ class Flasher {
         const messageToken = this._sendChunk(chunkIndex);
 
         // We don't need to wait for the response if using FastOTA.
-        if (!canUseFastOTA) {
+        if (canUseFastOTA) {
```

Comment:
Wouldn't this be a bug if it was `!canUseFastOTA`? Seems like this wouldn't work before...